### DISCUSSION TITLE
Use new [SerializeSyncType] Attribute that auto combines [AllowMutableSyncType] and [SerializeField]

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#526] [Feature Request]
Is it possible to make an attribute called "[SerializeSyncType]" that automatically puts [AllowMutableSyncType] and [SerializeField] in one Attribute?

**Comment by FirstGearGames** [comment]
Not sure if I can override the SerializeField one, but this seems like a QOL feature that is more trouble than it's worth TBH.

**Reply by Katerlad** [reply]
Booooo 😛
